REFERENCE.md

```diff
@@ -834,6 +834,33 @@ Functionality can be tested with cipherscan or similar tool: https://github.com/
 
 Default value: []
 
+##### `ssl_crl_check`
```

Comment:
it's Ok to add these here, but they _should_ get regenerated when we build the strings before release.

Reply:
didn't notice that this is generated, removed this change